ts/Core/Axis/Color/ColorAxis.ts

```diff
@@ -572,13 +573,29 @@ class ColorAxis extends Axis implements AxisLike {
         ).attr({
             zIndex: 1
         }).add(item.legendGroup);
-
+        const options = legend.options;
+        const symbolPadding = options.symbolPadding || 0;
+        const isSeries = !(item as any).series;
```

Comment:
The `item` is never a series. `series` property on `item` is an array of series because the `item` is an axis - colorAxis to be precise.